arch/x86/kernel/slaunch.c

```diff
@@ -27,13 +27,72 @@
 #include <asm/e820/api.h>
 #include <asm/bootparam.h>
 #include <asm/setup.h>
+#include <linux/fs.h>
 #include <linux/slaunch.h>
 
+#define MSG_BUFFER_LEN 20
+#define MSG_ERROR "Mapping Error\n"
+
 #define PREFIX	"SLAUNCH: "
 
+#define DECLARE_PUB_READ(reg_name, reg_offset, reg_size)					\
```

Comment:
Please move all this down to be colocated with the rest of the securityfs code.

Reply:
Once you co-locate all your securityfs related changes, they will go with commit 5544017b55129000fb8ada14c09a7cfcfc47be62